spec/support/disability_compensation_form/all_claims_fe_submission.json

```diff
@@ -114,7 +114,6 @@
         }
       ]
     },
-    "standardClaim": false,
```

Comment:
is standardClaim optional?

Reply:
So, what I was going to say is: The data processing looks like FE => BE form translation => EVSS.`standardClaim` is not always passed from the FE to the BE.  So this PR is to force the value to `false` if it was not provided in the FE data.  EVSS requires a value in some circumstances.
But now that I'm looking at the form, the question that creates the data for this field is required. (See screenshot below.) I'm going to take out the changes that have to do with setting a default value for `standardClaim` because I think it's redundant.

![Screen Shot 2020-07-07 at 10 07 33 AM](https://user-images.githubusercontent.com/9893700/86842208-7af27d80-c06a-11ea-8a66-c7b17c49331f.png)

Reply:
Sounds good, just tag as Ready for Review when you're all set.